ims/ostis_tech/justification/section_type_function_and_composition_of_technology_OSTIS/section_type_function_and_composition_of_technology_OSTIS.scs

```diff
@@ -9,10 +9,3 @@ section_composition_of_OSTIS_technology
 
 	section_composition_of_OSTIS_technology = [*^"file://section_type_function_and_composition_of_technology_OSTIS.scsi"*];;
 
-rrel_key_sc_element -> (section_composition_of_OSTIS_technology -> technology_OSTIS)
-(*
-
-	=> nrel_key_sc_element_order: (complex_technology_development_of_intelligent_systems -> technology_OSTIS)
```

Comment:
Technology OSTIS should stay a key sc-element for section_composition_of_OSTIS_technology but not for complex_technology_development_of_intelligent_systems